sentry/src/main/java/io/sentry/SentryMeasurementUnit.java

```diff
@@ -0,0 +1,111 @@
+package io.sentry;
+
+import java.util.Locale;
+import org.jetbrains.annotations.ApiStatus;
+import org.jetbrains.annotations.NotNull;
+
+/**
+ * <a href="https://getsentry.github.io/relay/relay_metrics/enum.MetricUnit.html#">Develop Docs</a>
+ */
+public interface SentryMeasurementUnit {
```

Comment:
`h`: Can we rename this? We also have `MeasurementValue` and not `SentryMeasurementValue`. Furthermore, I don't think there will be many naming clashes.
```suggestion
public interface MeasurementUnit {
```

Reply:
tbh I'm fine with both, just wanted to keep it aligned with the dart implementation. But I see now js also calls it just `MeasurementUnit`